wireless_modules/middle_module/test.py

```diff
@@ -0,0 +1,51 @@
+# Author: Kunj
+# Last Modified: 3/08/2020
```

Comment:
Git will keep track of who and when modified this code. I don't think we need to explicitly include this